next-sitemap.config.js

```diff
@@ -4,9 +4,30 @@
  */
 
 /** @type {import('next-sitemap').IConfig} */
+const siteUrl = process.env.SITE_URL || 'https://developer.hashicorp.com'
+
 module.exports = {
-	siteUrl: process.env.SITE_URL || 'https://developer.hashicorp.com',
-	generateRobotsTxt: true, // (optional)
-	exclude: ['/swingset*', '/onboarding/*', '/profile*'],
+	siteUrl,
+	generateRobotsTxt: true,
+	robotsTxtOptions: {
+		additionalSitemaps: [
+			`${siteUrl}/hcp-sitemap.xml`,
+			`${siteUrl}/sitemap-docs.xml`,
+		],
+	},
+	exclude: [
+		'/swingset*',
+		'/onboarding/*',
+		'/profile*',
+		'sitemap.xml',
+		'/boundary*',
+		'/consul*',
+		'/nomad*',
+		'/packer*',
+		'/terraform*',
+		'/vagrant*',
+		'/waypoint*',
+		'/vault*',
```

Comment:
by excluding these, are we now missing out on sitemap entries that we were previously generating? Ex. anything "tutorials" related:

<img width="929" alt="CleanShot 2023-02-22 at 17 44 18@2x" src="https://user-images.githubusercontent.com/26389321/220777319-21f30869-d619-4eb2-b0f4-074eddd17e22.png">

(@ https://developer.hashicorp.com/sitemap-0.xml)

---

It looks like the new sitemap only has HCP tutorials, which seems like a undesired regression here:
- https://dev-portal-git-na-create-docs-content-sitemap-hashicorp.vercel.app/sitemap-0.xml

Reply:
In the theme of incremental adoption, maybe these `exclude` values can be narrowed down a bit, like:

```diff
- /waypoint*
+ /waypoint/docs*
+ /waypoint/commands*
+ // etc... I.e. only exclude "docs"
```

This way we're only excluding docs, since they're now coming from your new API endpoint, but we're still using dev-portal's build process for tutorials, integrations, etc.

Reply:
added